libs/dh/esett/feature-balance-responsible/src/lib/drawer/dh-drawer.component.html

```diff
@@ -14,69 +14,67 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<ng-container *transloco="let t; read: 'eSett.balanceResponsible.shared'">
-  <watt-drawer #drawer (closed)="onClose()">
-    <watt-drawer-heading>
-      <h2 class="message-heading">{{ balanceResponsibleMessage?.supplierWithName?.value }}</h2>
+<watt-drawer
+  *transloco="let t; read: 'eSett.balanceResponsible.shared'"
+  #drawer
+  [autoOpen]="true"
```

Comment:
suggestion: when `autoOpen` is set, a `key` input must be set as well

Reply:
Thank you, forgot about that one

Reply:
Fixed